server/server-utils/emailProvider.js

```diff
@@ -0,0 +1,34 @@
+var nodemailer = require('nodemailer');
+
+function emailProvider() {
+
+  //define environment variables for e-mail
+  if (!process.env.APP_EMAIL || !process.env.APP_EMAILPW) {
```

Comment:
Moving the checks to server.js so that it checks at startup and crashes if in production mode and the environment variables aren't defined.